xssp_rest/tasks.py

```diff
@@ -94,6 +94,14 @@ def mkhssp_from_sequence(sequence, output_format):
 
     If present, the input FASTA description line is used.
     """
+
+    if output_format == 'hssp_stockholm':
+        try:
+            output = get_hg_hssp(sequence)
+            return output
+        except RuntimeError as e:
+            _log.info("hg-hssp content cannot be acquired, continuing without")
```

Comment:
The log message should say something like "No cached data found. Data will be generated now.".